src/app/auth/auth.component.html

```diff
@@ -0,0 +1 @@
+<app-register></app-register>
```

Comment:
why are you adding register component here?
this component is not used please delete it